src/foam/u2/view/UnstyledTableView.js

```diff
@@ -123,6 +123,12 @@ foam.CLASS({
         model of the object being shown in the table.
       `
     },
+    {
+      class: 'Boolean',
+      name: 'inheritModelActions',
```

Comment:
When would we not want to do this?

Reply:
Sub classes - In context of a User, if I subclass User to let's say a Contact and display those contacts within a table view, I may not want to show the actions of the User class.

Reply:
Is this a real or hypothetical requirement?

Reply:
Let me reiterate since I don't believe I explained this change properly.

The tableview defaults to including its data model actions _**and**_ contextMenuActions defined on the table view instance. This property changes that behaviour by dictating whether those should be combined or not. I could choose to opt out of my table view using its data model actions and solely use the contextMenuActions provided in the tableview instance.

I did it for a reason but I can't recall why since this was a ways back.

Reply:
So, it isn't really "inherit"ModelActions but "user"ModelActions?

Reply:
useModelActions, right.

Reply:
or how about utilizeModelActions HAH